src/components/sections/launch-sequence.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Countdown from 'react-countdown';
+import SecondaryCta from '../secondary-cta';
+
+const hbpStartDate = new Date('12/31/2021, 23:59:59').getTime();
+let today = new Date();
+today = Date.now();
+const diff = hbpStartDate - today;
+
+const time = ({days, hours, minutes}) => {
+  var minutePadding = "";
+  var hourPadding = ""
+  if (minutes < 10) {
+    minutePadding = 0;
+  }
+  if(hours < 10) {
+    hourPadding = 0;
+  }
+  return <span>{days} : {hourPadding}{hours} : {minutePadding}{minutes} </span>;
```

Comment:
There should be a method String.padStart that you can use

Reply:
fixed :D